lib/ruby-stackoverflow/client/resources.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'resources/resource'
+require_relative 'resources/user'
+require_relative 'resources/question'
+require_relative 'resources/answer'
+require_relative 'resources/notification'
+require_relative 'resources/badge'
+require_relative 'resources/reputation'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.